src/app.module.ts

```diff
@@ -127,9 +129,9 @@ export class AppModule implements NestModule {
         TypeOrmModule.forRootAsync({
           imports: [ConfigModule],
```

Comment:
This can be done in a separate PR, but we though theoretically only use `@/config/configuration.module` here as well (if possible).

Reply:
Unfortunately, Since we need `env` variables, it's not possible to use `configuration.module`.